wled00/wled.cpp

```diff
@@ -8,6 +8,90 @@
 #include "soc/rtc_cntl_reg.h"
 #endif
 
+#ifdef WLED_USE_ETHERNET
```

Comment:
This should stay in its separate file, `wled.cpp` is already quite cluttered and should maybe itself be split at some point. We can add `wled_ethernet.cpp` if required.

Reply:
Yes, there are a few functions and perhaps globals that would be petfect fit for fcn.cpp.